spec/support/context/with_certificate.rb

```diff
@@ -1,10 +1,17 @@
+require 'spec_helper_acceptance'
+
 shared_context 'with certificate copied to system under test' do
   before(:all) do
     @client = AzureHelper.new
     @template = 'azure_vm_classic.pp.tmpl'
 
     @local_private_key_path = File.join(Dir.getwd, 'spec', 'acceptance', 'fixtures', 'insecure_private_key.pem')
-    @remote_private_key_path = '/tmp/id_rsa'
+
+    if default['platform'] =~ /^windows/
```

Comment:
`default` is only present when running under beaker, so this fails here is run with PUPPET_AZURE_BEAKER_MODE=local.

I'd reformulate so as to default to /tmp/id_rsa unless `default` defined AND `default['platform']` is windows.